src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -676,4 +678,16 @@ private void UpdateShownPatchDetails()
         // Enable move to patch button if this is a valid move
         moveToPatchButton.Disabled = !editor.IsPatchMoveValid(patch);
     }
+
+    private void OnLineEditTextChanged(string new_text)
```

Comment:
This should have much more descriptive name. Like `OnSpeciesNameTextChanged` and the parameter case should be `newText` instead of snake_case